### DISCUSSION TITLE
Updating from v10.6.1 to v11.2 failes

### DISCUSSION THREAD

**Discussion by kalwados** [jomjol/AI-on-the-edge-device#1017] [Q&A] [answered]
Yesterday I tried to upgrade the software on the ESP32Cam to latest version 11.2.0. I've uploaded fireware.bin and html.zip as described in the wiki. After pressing "flash firmware" the device did not go online again. After a power off/on cycle there is still v10.6.1 active. 
Since the firmware.bin is now on the SDCard (checked hash values), i tried `http://192.168.0.x/ota?file=firmware.bin` with no success. After a reboot v10.6.1 is still active. What am I doing wrong?

**Comment by jomjol** [comment]
Well hard to say, sometimes it goes wrong for no reasonable reason. Did you try it again after the reboot`?

**Reply by kalwados** [reply]
I've tried that several times without success. Still V10.6.1 running. The ESP32 is still plugged into the programming board like this one: https://www.berrybase.de/esp32-cam-mb-usb-zu-seriell-schnittstelle-fuer-esp32-cam. 
Any known problems with that?

**Comment by jomjol** [comment, marked as answer]
If nothing else helps you need to do a very hard reset:
1) Erease the flash memory
2) Flash the new software

Basically a new installation of the firmware.

**Reply by kalwados** [reply]
Yes, that worked. Will try OTA update again when v11.3 is available ;-)
